eth-contracts/contracts/ServiceProviderFactory.sol

```diff
@@ -117,15 +117,17 @@ contract ServiceProviderFactory is InitializableV2 {
      * @notice Function to initialize the contract
      * @param _governanceAddress - Governance proxy address
      */
-    function initialize (address _governanceAddress) public initializer
+    function initialize (
+        address _governanceAddress,
+        uint _decreaseStakeLockupDuration
+    ) public initializer
     {
         governanceAddress = _governanceAddress;
 
         // Configure direct minimum stake for deployer
         minDeployerStake = 5 * 10**uint256(DECIMALS);
 
-        // 10 blocks for lockup duration
-        decreaseStakeLockupDuration = 10;
+        decreaseStakeLockupDuration = _decreaseStakeLockupDuration;
```

Comment:
Note - currently stored as a `uint` instead of `uint256` which needs to change, but there is a separate issue for that under `Notes` so I'm leaving this to be handled separately